Student/Student/Submissions/SubmissionDetails/SubmissionDetailsPresenter.swift

```diff
@@ -147,7 +147,7 @@ class SubmissionDetailsPresenter: PageViewLoggerPresenterProtocol {
 
     func select(drawerTab: Drawer.Tab?) {
         selectedDrawerTab = drawerTab ?? .comments
-        view?.embedInDrawer(viewControllerForDrawer())
+        update()
```

Comment:
We have a check in `update()` that will update the drawer if the drawer tab has changed. Using `update` avoids a race condition in the test.
```
func update() {
        ...
        let drawerTabChanged = selectedDrawerTab != currentDrawerTab
        if drawerTabChanged {
            view?.embedInDrawer(viewControllerForDrawer())
        }
```

Reply:
Will this cause the submission content to blink too, since it re-embeds?

Reply:
I haven't noticed any blinking. I think it's still doing the same number of embeds as before.

Reply:
k.